### PR TITLE
Index out of range fix

### PR DESCRIPTION
**Pull request by RealHandy**
Hi! Here's the PR for the issue #12 -- I see now you made related changes a few days ago, so maybe it makes sense that this change can be merged.

### CODE REVIEW

File: py_src/yolov4/tflite/__init__.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def load_tflite(self, tflite_path):
         self.input_size = input_details["shape"][1]
         self.input_index = input_details["index"]
         output_details = self.interpreter.get_output_details()
-        self.output_index = [output_details[i]["index"] for i in range(3)]
+        self.output_index = [output_details[i]["index"] for i in range(len(output_details))]
```
Comment (hhk7734) [inline py_src/yolov4/tflite/__init__.py]:
```python
        self.output_index = [details["index"] for details in output_details]
```

File: py_src/yolov4/tflite/__init__.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ def predict(self, frame: np.ndarray):
         self.interpreter.set_tensor(self.input_index, image_data)
         self.interpreter.invoke()
         candidates = [
-            self.interpreter.get_tensor(self.output_index[i]) for i in range(3)
+            self.interpreter.get_tensor(self.output_index[i]) for i in range(len(self.output_index))
```
Comment (hhk7734) [inline py_src/yolov4/tflite/__init__.py]:
```python
        candidates = [
            self.interpreter.get_tensor(index) for index in self.output_index
        ]
```

**Review by hhk7734 — CHANGES_REQUESTED** [review]
I would like to change it like this

**Comment by hhk7734** [comment]
Your suggestion looks good! :)

File: py_src/yolov4/tflite/__init__.py (hunk 1 above)
Comment (RealHandy) [reply py_src/yolov4/tflite/__init__.py]: Of course! that's way better.

File: py_src/yolov4/tflite/__init__.py (hunk 2 above)
Comment (RealHandy) [reply py_src/yolov4/tflite/__init__.py]: Yeah, that's much better. I'm a python noob. Thanks!

**Pull request merged**